libs/documentation/src/lib/components/search/demos/optional/search-optional.component.html

```diff
@@ -1,25 +1,25 @@
 <div>
     <h4>Search with dropdown</h4>
     <p> <code> dropdown: {{'{'}} options: []{{' }'}}</code> is used for selecting search with dropdown mode.</p>
-    <sds-search [(ngModel)]="ddmodel" [searchSettings]="ddSearchSettings" placeholder="Entity an entity ID, name, or keyword"></sds-search>
```

Comment:
Does the ts file already have the same placeholder as the one used here? If not, could you update the settings in the ts file as well.

Reply:
The ts has a different placeholder value, but it is the value that has always shown in the component. The one in the HTML was never patched through, and only visible when examining the HTML.